client/state/notices/actions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import {
+	NEW_NOTICE,
+	REMOVE_NOTICE
+} from 'state/action-types';
+
+import { uniqueId } from 'lodash';
```

Comment:
@gziolo, this is not a solution, because you would need globally uniqueId of the env to pass to uniqueId :)

But, behold:
https://github.com/Automattic/wp-calypso/pull/1793